### DISCUSSION TITLE
Parameter for GenerateStaticClasses

### DISCUSSION THREAD

**Discussion by Yahya0007** [shinyorg/shiny#974] [Q&A] [answered]
Hi

Shiny.GenerateStaticClasses now has a namespace parameter in the latest release. What value goes into this parameter please?

Thanks

Regards

**Comment by aritchie** [comment, marked as answer]
The namespace that you want to use

**Comment by Yahya0007** [comment]
Hi

 

Thanks.  I have used [assembly: Shiny.GenerateStaticClasses("StaffApp.Droid")] in Android project but static classes don’t seem to be generating now and I am getting errors on previously working shiny methods. Example;

 

'MainActivity' does not contain a definition for 'ShinyOnCreate' and no accessible extension method 'ShinyOnCreate' accepting a first argument of type 'MainActivity' could be found (are you missing a using directive or an assembly reference?)         StaffApp.Android

 

Thanks again for taking the time to reply.

 

Regards

 

Yahya

**Reply by aritchie** [reply]
I have no idea as you haven't given any details at all.  You have to file an issue with a reproducible sample as this is not the place for bugs.
